pag-parser/src/lib.rs

```diff
@@ -281,6 +281,7 @@ pub fn generate_parser(input: &str) -> Result<TokenStream, Error> {
             unreachable_patterns,
             unreachable_code,
             unused_assignments,
+            unused_parens,
```

Comment:
What is this for?

Reply:
https://github.com/SchrodingerZhu/paguroidea/pull/58#pullrequestreview-1483466121

Reply:
should be fixed in #61